modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -10,35 +10,109 @@ You can share subnets within a configured VPC with another AWS user account if t
+                "route53:GetChange",
+                "route53:GetHostedZone",
+                "route53:ListTagsForResource",
+                "route53:UpdateHostedZoneComment",
+                "tag:GetResources",
+                "tag:UntagResources"
+            ],
+            "Resource": "*"
+        }
+    ]
 }
+EOF
+----
++
+.. Create the policy in AWS:
++
+[source,terminal]
+----
+$ aws iam create-policy \
+    --policy-name SharedVPCPolicy \
+    --policy-document file:///tmp/shared-vpc-policy.json
+----
++
+You will attach this policy to a role necessary for the shared VPC permissions.
++
+. Create a custom trust policy file that grants permission to assume roles:
++
+[source,terminal]
+----
+$ cat <<EOF > /tmp/shared-vpc-role.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Account-ID>:root"  <1>
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+EOF
+----
++
+--
+<1> The principal will be scoped down later in this process after the *cluster-creating AWS account* user has created the necessary cluster roles. On creation, you must create a root user placeholder by using the *cluster-creator's AWS Account* ID as `arn:aws:iam::{Account}:root`.
+--
++
+. Create the IAM role:
++
+[source,terminal]
+----
+$ aws iam create-role --role-name <role_name> \  <1>
+    --assume-role-policy-document file:///tmp/trust-policy.json
 ----
 +
 --
-<1> The following principals are be added later in this process after the *cluster-creating AWS account* user has created these roles. On creation, you must create a root user placeholder by using the *cluster-creator's AWS Account* ID as `arn:aws:iam::{Account}:root`.
+<1> Replace _<role_name>_ with the name of the role you want to create.
 --
-    * The `ResourceGroupandTagEditorFullAccess` permissions policy
-    * The `Route53minimalPermissions` permissions policy
 +
-After you create this IAM role, provide the created role's ARN to the cluster creator.
+. Attach the custom `SharedVPCPolicy` permissions policy:
++
+[source, terminal]
+----
+$ aws iam attach-role-policy --role-name <role_name> --policy-arn \  <1>
+    arn:aws:iam::<AWS_account_ID>:policy/SharedVPCPolicy  <2>
+----
++
+--
+<1> Replace _<role_name>_ with the name of the role you have created.
+<2> Replace _<AWS_account_ID>_ with the *VPC-owning AWS account* ID.
+--
++
+. Attach the AWS managed link:https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ResourceGroupsandTagEditorFullAccess.html[`ResourceGroupsandTagEditorFullAccess`] permissions policy:
++
+[source,terminal]
+----
+$ aws iam attach-role-policy --role-name <role_name> --policy-arn \
+    arn:aws:iam::aws:policy/ResourceGroupsandTagEditorFullAccess
+----
++
+After you create this IAM role and attach all the necessary policies, provide the created role's ARN to the *cluster-creating AWS account* user.
```

Comment:
Should this be a step between the step to attach the permissions policy and the step to create a Resource Share?

Reply:
No, this is a separate step.